feature1/one-more-file.txt

```diff
@@ -0,0 +1 @@
+This file was added in feature1 branch.
```

Comment:
It's really good you added this line of code

Reply:
Thanks for this comment.